packages/harmony/src/components/button/SocialButton/SocialButton.tsx

```diff
@@ -48,13 +46,26 @@ const getSocialButtonProps = (type: SocialMedia): ButtonProps => {
 }
 
 export const SocialButton = (props: SocialButtonProps) => {
-  const { socialType, className, ...rest } = props
+  const { socialType, ...rest } = props
   const socialButtonProps = getSocialButtonProps(socialType)
+
+  const instagramBackgroundCss: CSSObject = {
+    background: `radial-gradient(61.46% 59.09% at 36.25% 96.55%, #ffd600 0%, #ff6930 48.44%, #fe3b36 73.44%, rgba(254, 59, 54, 0) 100%),
+      radial-gradient(202.83% 136.37% at 84.5% 113.5%, #ff1b90 24.39%, #f80261 43.67%, #ed00c0 68.85%, #c500e9 77.68%, #7017ff 89.32%)`
+  }
+
   return (
     <Button
       {...socialButtonProps}
       {...rest}
-      className={cn(styles.root, socialButtonProps.className, className)}
+      css={({ color }) => ({
+        border: 'transparent',
+        ...(socialType === 'instagram' && instagramBackgroundCss),
+
+        '& svg path': {
```

Comment:
Nice. Potentially can pass color prop into the icon itself?

Reply:
there is no static white color option for color on the icons and passing in fill was not working so leaving as is for now

Reply:
Let's add static white as an option cause I'd like us to not select svgs directly

Reply:
would need to update the "icon" section in semantics to have static-white, and then just need to add `color='staticWhite` to icon and should work. might be worth checking with sammie on how she'd like us to handle static-white icons, like should that not be an option directly on the icon and come from a parent override?